pandas/tests/io/pytables/test_store.py

```diff
@@ -4230,9 +4223,8 @@ def do_copy(f, new_f=None, keys=None, propindexes=True, **kwargs):
 
                         fd, new_f = tempfile.mkstemp()
 
-                    tstore = store.copy(
-                        new_f, keys=keys, propindexes=propindexes, **kwargs
-                    )
+                        tstore = store.copy(
+                            new_f, keys=keys, propindexes=propindexes, **kwargs)
```

Comment:
You'll need to run `black` on any file you've changed, see https://pandas.pydata.org/pandas-docs/dev/development/contributing.html#python-pep8-black

Reply:
Thanks for this, just done this.